src/app/Http/Controllers/PostController.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Http\Requests\PostRequest;
+use App\Models\Post;
+
+class PostController extends Controller
+{
+    const DESTINATION_PATH = 'public/images';
+
+    public function store(PostRequest $request)
+    {
+        $post = new Post();
+        $post->content = $request->input('content');
+        $post->user_id = auth()->id();
+
+        if ($request->hasFile('image')) {
+            $image = $request->file('image');
+            $image_name = $image->getClientOriginalName();
+            $path = $image->storeAs(self::DESTINATION_PATH, $image_name);
+
+            $input['image'] = $image_name;
+            $post->image = $path;
+        }
+
+        $post->save();
+        $returnRoute = $request->input('return_route', 'user.home');
```

Comment:
what is this?

Reply:
after creating post po sir mag r'redirect po sa same page.

Reply:
why it is in the $request->input. just wondering

Reply:
You can use redirect back withInput

https://laravel.com/docs/10.x/redirects

Reply:
based po sa nabasa ko sir inaattempt po niyan na iretrieve yung value na associated sa return_route galing po sa request's input data. if nag-eexist daw po yung data, yun po yung gagamitin na value.

Reply:
I have resolved this comment, sir. Thank you! 🙇